determined-ai/determined
# feat: introduce logStreamActor & add notebook logs endpoint [DET-4100]

## Description

- define helpers around handling logs for our APIs
- define and use LogEntry message for the API response
- add logStreamActor to handle streaming for logs.
- add notebook streaming log endpoint



## Test Plan

- create a notebook and listen for logs. 
- make sure the request is closed when the notebook is terminated
- added and ran test.sh




## Commentary (optional)




## Checklist

- [ ] remove temporary test files
See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234
